metadata/datasets.yaml

```
@@ -579,16 +579,16 @@
     short_name: SpeechPref
     key: 1Nc7X0zNUAwKyXJro--MbGiXKrM5NlkYR10Y3KJd-N_c
     filename: nat_speech_pref
-    citation: "Issard & Cristia (in prep.)"
+    citation: "Issard et al. (2020)"
     internal_citation: ""
```

Missing internal citation, see other datasets
```suggestion
    internal_citation: "(Issard et al., 2020)"
```